package/yast2-installation-control.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Dec 14 11:50:09 UTC 2018 - dgonzalez@suse.com
+
+- Add macros related to skelcd (fate#325482)
```

Comment:
@lslezak right now, I am not able to think in a less technical log entry. Ideas are very welcome :)

Reply:
That's OK, it's a technical detail so it cannot be described easily. Moreover this package is intended for developers, not for regular end users so it does not matter in this case.